src/components/Experience.tsx

```diff
@@ -31,28 +26,27 @@ const experienceDetails: ExperienceDetail[] = [
 ];
 
 const Experience = () => {
-  const [currentExperienceDataIndex, updateExperienceDataIndex] = useState<
-    undefined | number
-  >(undefined);
+  const [currentExperience, setExperience] = useState<undefined | IExperience>(
```

Comment:
Again the same convention issue. The convention says that the state updater function should be "setCurrentExperience".

I would suggest installing Sonar lint VS Code extension. It will identify these kind of issues beforehand.